LICENSE.txt

```diff
@@ -1,16 +1,59 @@
-MIT License
+Modified Autonomous Software License (MASL)
```

Comment:
Because GPL is too restrictive.

https://github.com/chaosbot/chaos/issues/220#issuecomment-303960463